packages/stark-ui/src/modules/table/components/column.component.ts

```diff
@@ -169,27 +179,26 @@ export class StarkTableColumnComponent extends AbstractStarkUiComponent implemen
 	 * Reference to the MatColumnDef embedded in this component
 	 */
 	@ViewChild(MatColumnDef)
-	public columnDef: MatColumnDef;
+	public columnDef!: MatColumnDef;
 
 	/**
 	 * Reference to the transcluded template in this component via the ngTemplateOutlet
 	 */
 	@ContentChild(TemplateRef)
-	public columnTemplate: TemplateRef<object>;
+	public columnTemplate?: TemplateRef<object>;
 
 	/**
 	 * @ignore
 	 * Internal formControl to manage the filter value of the column
 	 */
-	public _filterFormCtrl: FormControl;
+	public _filterFormCtrl: FormControl = new FormControl();
```

Comment:
I think here is better to use the `!` modifier because we know we will instantiate a new formControl in the `ngOnInit`. Otherwise we will also instantiate a dummy formControl which will also consume some memory and at the end it will never be used.

Reply:
Or I remove it from the onInit an change this to `	public _filterFormCtrl: FormControl = new FormControl(this.filterValue);`?